Docs Patch to include Kubernetes Installation (#7075)
hashicorp/consul: website/source/docs/install/index.html.md

# Install Consul

> Installing Consul is simple. There are two main approaches to installing Consul:

Reviewer: ```suggestion
There are two main approaches for installing Consul:
```

1. [Installing on Kubernetes](#install-on-kubernetes) 

> 1. Installing as a standalone binary 

Reviewer: Could you add the link back? `(#precompiled-binaries)`

Author: This is included as a sub bullet link below.

   a. Using a [precompiled binary](#precompiled-binaries)

   b. Installing [from source](#compiling-from-source)

>    Downloading a precompiled binary is easiest, and we provide downloads over TLS

Reviewer: ```suggestion
   Downloading a precompiled binary is quickest and we provide downloads over TLS
```

   along with SHA256 sums to verify the binary. We also distribute a PGP signature
   with the SHA256 sums that can be verified.

The Getting Started guides provide a quick walkthrough of installing and using Consul as a

* [Standalone binary your local machine](https://learn.hashicorp.com/consul/getting-started/install?utm_source=consul.io&utm_medium=docs). 
* [DaeemonsSet on Minikube on your local machine](https://learn.hashicorp.com/consul/kubernetes/minikube?utm_source=consul.io&utm_medium=docs&utm_content=k8s&utm_term=mk)

## Install on Kubernetes

> Installing Consul on Kubernetes is done through Helm, which provides an easy way to deploy applications on Kubernetes. 

Reviewer: ```suggestion
Installing Consul on Kubernetes is done through Helm, which provides a convenient way to deploy applications on Kubernetes. 
```

Determine the latest version of the Consul Helm chart

> by visiting [https://github.com/hashicorp/consul-helm/releases](https://github.com/hashicorp/consul-helm/releases).

Reviewer: We don't typically use full urls in docs.

Author: This was copied and pasted from the Kubernetes install docs, how would you link to a Github releases page from this doc?

Reviewer 2: We shouldn't be copying this doc. Just link to the existing install guide. Otherwise we have to update things in multiple places which inevitably we'll forget to do.

Clone the chart at that version. For example, if the latest version is
`v0.8.1`, you would run:

```bash
$ git clone --single-branch --branch v0.8.1 https://github.com/hashicorp/consul-helm.git
Cloning into 'consul-helm'...
...
You are in 'detached HEAD' state...
```

Now you're ready to install Consul! To install Consul with the default
configuration using Helm 3 run:

```sh
$ helm install hashicorp ./consul-helm
NAME: hashicorp
...
```

> A more detailed set of instructions can be found [here](/docs/platform/k8s/run.html). 

Reviewer: Can you link to both the docs and Learn? 
```suggestion
A more detailed set of instructions can be found [here](/docs/platform/k8s/run.html). 

```
